content/docs/languages/rust/40-supply-chain-analysis.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Supply chain analysis"
+slug: rust-supply-chain-analysis
+summary: "This section describes tricks for Rust unit testing"
+weight: 40
+---
+
+# Supply chain analysis
+
+## Vetting
+
+The tools in this section are more for "understanding" than "checking." E.g., running them does not produce "bug reports", but can help you assess maturity and security of dependencies. Tools below are rather "quantitative" than "qualitative" - you will need to do manuall, in-depth review of the outputs to extract any solid evidences about the maturity.
+
+ Run cargo-supply-chain
+This reveals who you are implicitly trusting when you rely on a dependency (e.g., you want that set to be small)
+ Run cargo-vet
+This checks if dependencies were audited by a "trusted party"
+rust-crate-audits - collection of Google's audits
+ Run cargo-crev
+This is a distrubuted core-review platform
+ Run cargo-deny
+Cargo plugin for linting your dependencies
```

Comment:
From line 14 (containing "Run cargo-supply-chain") to this one, it looks like the text was meant to be formatted as lines/bullets. But it displays as a single paragraph:
<img width="742" alt="Screenshot 2024-06-14 at 1 04 38 PM" src="https://github.com/trailofbits/testing-handbook/assets/35515885/01981eb5-5cf0-42a3-b6b8-289160e4abe0">

Reply:
fixed